be/src/common/config.h

```diff
@@ -482,6 +482,9 @@ CONF_Int64(compaction_memory_limit_per_worker, "2147483648"); // 2GB
 CONF_String(consistency_max_memory_limit, "10G");
 CONF_Int32(consistency_max_memory_limit_percent, "20");
 CONF_Int32(update_memory_limit_percent, "60");
+// This is a soft limit, if primary index need more memory actually, the memory usage
+// may be greater than it in order to keep process run success
+CONF_mInt32(soft_update_memory_limit_percent, "20");
```

Comment:
No need to add another config for control the memory usage of primary key index, too hard to understand.